### PR TITLE
feat(VADC-261): add combined gwas steps template

### PR DESCRIPTION
**Pull request by alilahrime**
### New Features
beginning steps template combined gwas

### CODE REVIEW

File: src/Analysis/GWASV2/GWASContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            title: 'Step 1',
+            description: 'Select Study Population',
+        },
+        {
+            title: 'Step 2',
+            description: 'Select Outcome Phenotype',
+        },
+        {
+            title: 'Step 3',
+            description: 'Select Covariate Phenotype',
```
Comment (pieterlukasse) [inline src/Analysis/GWASV2/GWASContainer.jsx:18]: I think "Select Covariate Phenotypes" will be better, as the user can select multiple covariate phenotypes here

File: src/Analysis/GWASV2/GWASContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        },
+        {
+            title: 'Step 4',
+            description: 'Review and submit your GWAS study',
```
Comment (pieterlukasse) [inline src/Analysis/GWASV2/GWASContainer.jsx:22]: Not sure where this new description is coming from. The wireframe seems to still state "Configure GWAS" https://xd.adobe.com/view/5773a0b3-0957-443b-830b-95318a30363c-0220/screen/b66dc7ca-840f-4cc9-92ba-0e02b21319fb

File: src/localconf.js [hunk 3]
```diff
@@ -344,98 +344,105 @@ function buildConfig(opts) {
...
-            description: 'Use this App to view status & results of submitted workflows',
-            image: '/src/img/analysis-icons/gwasResults.svg',
-          };
-          break;
-        default:
-          break;
+              image: '/src/img/analysis-icons/virulence.png',
+            };
+            break;
+          case 'vaGWAS':
```
Comment (pieterlukasse) [inline src/localconf.js:366]: can we just remove this one?

**Review by pieterlukasse — COMMENTED** [review]
looks good! Just a few minor questions/suggestions

File: src/Analysis/GWASV2/GWASContainer.jsx (hunk 1 above)
Comment (alilahrime) [reply src/Analysis/GWASV2/GWASContainer.jsx:18]: true

File: src/Analysis/GWASV2/GWASContainer.jsx (hunk 2 above)
Comment (alilahrime) [reply src/Analysis/GWASV2/GWASContainer.jsx:22]: is this the catch-call for all remaining things including but not limited to submission?

Comment (alilahrime) [reply src/Analysis/GWASV2/GWASContainer.jsx]: edited

Comment (pieterlukasse) [reply src/Analysis/GWASV2/GWASContainer.jsx]: yes...for now 😅

File: src/localconf.js (hunk 3 above)
Comment (alilahrime) [reply src/localconf.js:366]: vaGWAS?

Comment (alilahrime) [reply src/localconf.js:366]: im not sure if its being used anywhere else. probably not? I think removing it should be part of a separate ticket/pr though

Comment (pieterlukasse) [reply src/localconf.js:366]: Ok, good point. Looks like it was added here https://github.com/uc-cdis/data-portal/pull/501/files. I'll add a ticket to the board.

**Review by pieterlukasse — APPROVED** [review]

File: src/localconf.js (hunk 3 above)
Comment (pieterlukasse) [reply src/localconf.js:366]: https://ctds-planx.atlassian.net/browse/VADC-228

**Pull request merged**